### DISCUSSION TITLE
Display package size

### DISCUSSION THREAD

**Discussion by prichodko** [rainbow-me/rainbowkit#326] [Feedback]
Congrats on the launch, the kit looks terrific.

One thing I am missing on the homepage / in the docs is information about the package size. I think it would be great to see how much `kB` is added initially and whether different options have any effect on it as well.

**Comment by nickbytes** [comment]
@prichodko thanks for the kind words! This is great feedback!

We've got this on the roadmap (+ exploring how we can decrease the bundle size). It's a tough situation right now, since some of the connector libraries are quite large.

**Reply by prichodko** [reply]
Yeah, I am aware of that. Though as a developer, I would prefer more transparency or maybe some warning.

**Reply by nickbytes** [reply]
How would you measure package size? Would you just measure RainbowKit? Would you measure RainbowKit + it's dependencies  (wagmi and ethers)?

**Reply by prichodko** [reply]
I would show separate sizes of RainbowKit (which I am sure is small) and its peer dependencies (which are big).
